osu.Game/Skinning/Editor/SkinEditorMenuBackgroundChooser.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Database;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterfaceV2;
+using osuTK;
+
+namespace osu.Game.Skinning.Editor
+{
+    public class SkinEditorMenuBackgroundChooser : Container, ICanAcceptFiles, IHasPopover
+    {
+        private readonly string[] handledExtensions = { ".jpg", ".jpeg", ".png" };
+
+        private readonly SkinEditor skinEditor;
+
+        private readonly Bindable<FileInfo?> menuBackground;
+
+        private MenuBackgroundChooserPopover? popover;
+
+        public bool PopoverVisible => popover?.State.Value == Visibility.Visible;
+
+        [Resolved(canBeNull: true)]
+        private OsuGame? game { get; set; }
+
+        public SkinEditorMenuBackgroundChooser(SkinEditor skinEditor)
+        {
+            this.skinEditor = skinEditor;
+
+            menuBackground = new Bindable<FileInfo?>();
+            menuBackground.BindValueChanged(onFileSelected);
+        }
+
+        private void onFileSelected(ValueChangedEvent<FileInfo?> file)
+        {
+            if (file.NewValue == null)
+                return;
+
+            this.HidePopover();
+            skinEditor.SetMenuBackground(file.NewValue);
+        }
+
+        public Task Import(params string[] paths)
+        {
+            Schedule(() =>
+            {
+                HoverClickSounds sounds = new HoverClickSounds();
+                AddInternal(sounds);
+                sounds.TriggerClick();
+                sounds.Expire();
```

Comment:
On an audio design level I don't think drag-and-drop operations should be emitting click sounds.